controllers/helmreleaseproxy/helmreleaseproxy_controller_test.go

```diff
@@ -438,3 +444,9 @@ func TestReconcileDelete(t *testing.T) {
 		})
 	}
 }
+
+func init() {
```

Comment:
Why is this needed in for the unit tests in this file?

Reply:
As unit-tests utilizing`fake.NewClientBuilder()` it requires sheme's to be regitered.